docs/app/services/playground/utilities/system-helper.ts

```diff
@@ -44,7 +44,7 @@ export class SystemJSHelper {
         { name: 'rxjs', path: 'rxjs@6.5.2' },
         { name: 'ngx-bootstrap', path: `ngx-bootstrap@${NGX_BOOTSTRAP_VERSION}/bundles/ngx-bootstrap.umd.min.js` },
         { name: 'chance', path: 'chance' },
-        { name: 'chart.js', path: 'chart.js/dist/Chart.bundle.min.js' },
+        { name: 'chart.js', path: 'chart.js/dist/chart.min.js' },
```

Comment:
Thanks! We haven't actually done any testing with 3.x yet, so probably better to clamp this to the version that the package depends upon.
```suggestion
        { name: 'chart.js', path: 'chart.js@2/dist/Chart.bundle.min.js' },
```

Reply:
Please check again. Changed to `chart.js@2.9.3` which is the version used in package-lock.json.